packages/cli/src/commands/build.ts

```diff
@@ -301,6 +308,145 @@ async function doBuild(
     ...pickOverrides(localConfig),
   };
 
+  const localFileSystem = new LocalFileSystemDetector(cwd);
+
+  const framework = await detectFramework({
```

Comment:
the name `framework` is is misleading, can we rename to `monorepoManager` instead?